sdk/samples/keyvault/keyvault/samples/src/keys_client_example.c

```diff
@@ -69,57 +66,67 @@
     } \
   } while (0)
 
-int exit_code = 0;
-
 az_span get_key_version(az_http_response* response);
 az_span const key_name_for_test = AZ_SPAN_LITERAL_FROM_STR("test-new-key");
 
+#ifdef _MSC_VER
+// "'getenv': This function or variable may be unsafe. Consider using _dupenv_s instead."
+#pragma warning(disable : 4996)
+// "Compiler will insert Spectre mitigation for memory load if /Qspectre switch specified"
+#pragma warning(disable : 5045)
+#endif // _MSC_VER
+
```

Comment:
It isn't great that we have to disable such warnings inside the samples. I don't have a great alternative though, atm.

Reply:
Other options were more verbose and were drawing too much attention, which wasn't good for samples. I don't like this either, and I tried to avoid this.